test/src/kbasesearchengine/test/main/NarrativeInfoDecoratorTest.java

```diff
@@ -121,12 +125,12 @@ public void searchObjectsDecorateWithNullInfo() throws Exception {
         final SearchObjectsInput dummyInput = new SearchObjectsInput()
                 .withPostProcessing(new PostProcessing().withAddNarrativeInfo(1L));
 
-        final List<ObjectData> objectdata = Arrays.asList(
+        final List<ObjectData> objectdata = new ArrayList<>(Arrays.asList(
```

Comment:
Why is the object data wrapped in another list? I don't see any code changes that would necessitate that.

Reply:
b/c originally I was suppose to set results to null. Arrays.asList didn't like me mutating the content

Reply:
Ok, can you put them back the way they were then? Or does that still cause problems?

Reply:
do I have to?

Reply:
One of the cardinal sins of SW development is adding unnecessary complexity so I'm going with yeah :P

Reply:
actually it does still cause problems as Arrays.asList won't let me remove items. https://stackoverflow.com/questions/7885573/remove-on-list-created-by-arrays-aslist-throws-unsupportedoperationexception